upgrading/upgrade-helm.adoc

```diff
@@ -15,6 +15,12 @@ If you have installed {product-title} by using Helm charts, to upgrade to the la
 * Update configuration files for the secured-cluster-services Helm chart.
 * Upgrade the secured-cluster-services Helm chart.
 
+[IMPORTANT]
+====
+* In order to ensure optimal functionality it is recommended to keep your secured clusters up to date and on the same version as Central. Out of date Sensor versions are supported for 6 months, but may not have the full functionality of an up to date version.
+* Enabling automated upgrades for your secured clusters is strongly recommended. Should you prefer manually upgrading your secured clusters refer to the xref:upgrade-roxctl.adoc#upgrade-secured-clusters[manual on upgrading secured clusters].
```

Comment:
I don't think that there are any automatic upgrades for Helm that are provided by ACS itself. This is something customers should arrange with their GitOps tools. I suggest removing this point.

```suggestion
```

Reply:
Fair, I'll change it.

But is there a reason why we don't support automatic upgrades using helm charts? Sounds like something we should do if possible.